bugs/PENDING_SUBMIT/0397-BAD-IMAGE-S20PLUS-41991e5f-no-opt-test-SM-G986B-R3CNA03E21W/reduced_manual/variant/shader.frag

```diff
@@ -0,0 +1,122 @@
+#version 320 es
+
+precision highp int;
+
+precision highp float;
+
+// Contents of injectionSwitch: [0.0, 1.0]
+layout(set = 0, binding = 0) uniform buf0 {
+ highp vec2 injectionSwitch;
+};
+precision highp float;
+
+precision highp int;
+
+struct struct_base {
+ int data;
+ int leftIndex;
+ int rightIndex;
+} ;
+
+struct_base struct_a[10] = struct_base[10](
+                            struct_base(1, - 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1),
+                            struct_base(1, 1, 1)
+                            );
+
+void func_a(inout struct_base struct_a, int input_a)
+{
+ struct_a.data = 1;
+}
+void func_b(int input_b, int input_a)
+{
+ int var_a = 0;
+ int var_b = 0;
+ // Always true. Iterates 6 times,
+ // but returns nothing on first iteration.
+ while(1 <= 1)
+  {
+   if(var_b >= 5)
+    {
+     break;
+    }
+   var_b ++;
+   // Clamp returns 0. Struct_a[0].leftIndex == -1.
+   // Always true.
```

Comment:
Nit but I would swap the order and avoid capitalizing Struct and Clamp. E.g.

// Always true.
// clamp(...) == 0
// struct_a[0].leftIndex == -1